modules/bootkube/resources/bootkube.sh

```diff
@@ -5,7 +5,7 @@ echo "Rendering Kubernetes core manifests..."
 
 # shellcheck disable=SC2154
 /usr/bin/docker run \
-    --volume "$(pwd)":/assets:z \
+    --volume "${PWD}:/assets:z" \
```

Comment:
Hmm, this may not play well with template replacement (e.g. see the `$$` [further down][1]).

[1]: https://github.com/openshift/installer/blob/ae7fe33afbf0bf3b19bf77b0ce26ffb4c98cfe11/modules/bootkube/resources/bootkube.sh#L90